csm/conf/setup.py

```diff
@@ -670,6 +670,48 @@ def _update_service_file(key, value):
             data = service_file_data.replace(key, value)
             Text(each_service_file).dump(data)
 
+    @staticmethod
+    def _set_healthmap_path():
+        """
+        This method gets the healthmap path fron salt command and saves the
+        value in csm.conf config.
+        """
+        minion_id = None
+        healthmap_folder_path = None
+        healthmap_filename = None
+        try:
+            """
+            Fetching the minion id of the node where this cli command is fired.
+            This minion id will be required to fetch the healthmap path.
+            Will use 'srvnode-1' in case the salt command fails to fetch the id.
+            """
+            minion_id = Setup.get_salt_data_with_exception(const.GRAINS_GET, const.ID)
```

Comment:
Suggestion, You should keep this in separate try catch block, as even if it fails, you want to proceed with default minion_id. But that's not case when it raise exception.

Reply:
Made the necessary changes. Now using the non-exception function and taken out the code from try-except block.